sklearn/tree/_export.py

```diff
@@ -133,6 +134,10 @@ def plot_tree(decision_tree, max_depth=None, feature_names=None,
     rotate : bool, optional (default=False)
         When set to ``True``, orient tree left to right rather than top-down.
```

Comment:
We should also change the docstring and say that it doesn't do anything, if it's not being used.